Options.lua

```diff
@@ -1592,10 +1592,7 @@ function opt:updateBWRaidList()
     local tree = aceOptions.args.bosses.args
 
     for k in next, loader:GetZoneMenus() do
-        if
```

Comment:
I'm not sure why this check was present.
Should this whole `if` be removed, or should we check for `BigWigs_BurningCrusade` in BCC?